Makefile

```diff
@@ -1,24 +1,35 @@
+_TEST_PROJECT = myproject
+
 start-openshift-with-catalog:
+	@echo "---- Starting OpenShift ----"
 	oc cluster up --service-catalog
-	oc login -u system:admin
-	oc adm policy add-cluster-role-to-user cluster-admin developer
-	oc login -u developer -p developer
-	oc project openshift
-	oc adm policy add-cluster-role-to-group system:openshift:templateservicebroker-client system:unauthenticated system:authenticated
-.PHONY: start-openshift-with-catalog
 
-start-openshift:
-	oc cluster up
+	@echo "---- Granting admin rights to Developer ----"
 	oc login -u system:admin
 	oc adm policy add-cluster-role-to-user cluster-admin developer
 	oc login -u developer -p developer
-	oc project openshift
-.PHONY: start-openshift
+
+	@echo "---- Switching to test project ----"
+	oc project $(_TEST_PROJECT)
+.PHONY: start-openshift-with-catalog
 
 stop-openshift:
 	oc cluster down
 .PHONY: stop-openshift
 
+_relist-template-service-broker:
```

Comment:
What is this for exactly?

Reply:
This is a new capability in OpenShift 3.7. It asks Template Service Broker to relist its content. In other words it refreshed Service Catalog entries.

Reply:
Ok. Maybe you should add your explanation as comment above.

Reply:
+1